app/views/posts/_new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h2>New blog post</h2>
+<%= form_with model: Post.new, url: "/users/#{current_user.id}/posts" do |form| %>
```

Comment:
- [ ] [OPTIONAL] on this line you can use the `@post` that you created in the [`new` method](https://github.com/Hombre2014/RoR-blog/blob/a27d9e925d6ee72e6c1850f72bc8c8332598c79f/app/controllers/posts_controller.rb#L12) instead of creating a new post to use as a model. 
```suggestion
<%= form_with model: @post, url: "/users/#{current_user.id}/posts" do |form| %>
```